java/common/GraqlToken.java

```diff
@@ -124,6 +124,7 @@ public static Filter of(String value) {
         SQUARE_OPEN("["),
         SQUARE_CLOSE("]"),
         QUOTE("\""),
+        SINGLE_QUOTE("'"),
```

Comment:
added single quote as requested

Reply:
can we rename this to `QUOTE_SINGLE`, and rename `QUOTE` to `QUOTE_DOUBLE`. Notice how they are both sortable on the prefix `QUOTE_` now @VladGan

Reply:
done